pkg/cmd/server/origin/master.go

```diff
@@ -454,6 +447,15 @@ func (c *MasterConfig) GetRestStorage() map[string]rest.Storage {
 		"clusterRoles":          clusterRoleStorage,
 	}
 
+	if !c.Options.DisabledFeatures.Has(configapi.FeatureBuilder) {
```

Comment:
can we move this check into a helper function like `BuildsEnabled` (see `WebConsoleEnabled`) and use it here and in start_master.go